### PR TITLE
Add tests for the weather module for each condition (cloud, wind, hum…

### PR DESCRIPTION
**Pull request by afbenedict**
…idity, temperature)

### CODE REVIEW

**Comment by CtrlC-Root** [comment]
Two minor formatting and style issues:

* Function docstrings should have a blank line below them.
* I would avoid breaking out function parameters into temporary variables if they're not used for anything else. If you wanted to be explicit you could pass them by name (`call_foo(a=1, b=2)`).

These are both examples of style issues that PEP8 won't catch but in my opinion should be standardized. Otherwise the unit tests pass so it looks good to me.

**Comment by CtrlC-Root** [comment]
:shipit:

File: src/brutus-module-weather/brutus_module_weather/tests/cloudQuestion_test.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+class CloudTestCase(BrutusTestCase):
+    """
+    Simple tests for the weather module
+    Check that questions about
+        clouds return the correct answers
+    """
+
+    cloudQuestions = ['Is it cloudy',
+                      'Are there a lot of clouds'
+                      'How many clouds are there'
```
Comment (sinasabet81) [inline src/brutus-module-weather/brutus_module_weather/tests/cloudQuestion_test.py]: Does python allow you to get away with not putting commas here?

Comment (afbenedict) [inline src/brutus-module-weather/brutus_module_weather/tests/cloudQuestion_test.py]: Good catch. I had already merged and deleted the branch so it is fixed in this pull request #39

**Pull request merged**